src/data/markdown/docs/03 cloud/01 Creating and running a test/04 Environment variables.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Environment variable'
```

Comment:
@MattDodsonEnglish 

I am not sure but perhaps `Environment Variables` and `Private Load Zones` would be better in another section - not at the beginning.

Reply:
I agree that they could be in a different category from the others, but I don't think we have a more appropriate sidebar title yet: at least these do pertain to "Running a test".

Another option would be to make a sidebar or nested section called something like "Security".

Reply:
My suggestion would be to place environment variables under **Manage** section, as it would stay consistent with the structure of the app (you can access env vars from that sub-menu). 

**Schedule a test** is also a part of **Manage** section, although one could argue that it should be a part of **Creating and running a test**.